### DISCUSSION TITLE
How does Fabric.js update a group's bounding box when its child objects are moved, especially if the group is rotated and interactive is set to true?

### DISCUSSION THREAD

**Discussion by turnbrain** [fabricjs/fabric.js#10097] [Q&A] [answered]
What function or mechanism in Fabric.js is responsible for updating the bounding box of a group after the positions of its child objects have been modified, particularly when the group is rotated and the group's interactive attribute is set to true?

[https://codepen.io/turnbrain/pen/KKjeBrb](https://codepen.io/turnbrain/pen/KKjeBrb)

**Comment by asturur** [comment, marked as answer]
It is the layoutManager instance for that group. You find it on group.layoutManager and there is a hook to execute it on the group that is group.performLayou()

**Reply by turnbrain** [reply]
Thanks
